src/SFA.DAS.AANHub.Application/Partners/CreatePartnerMemberCommand.cs

```diff
@@ -0,0 +1,39 @@
+﻿using MediatR;
+using SFA.DAS.AANHub.Application.Common.Commands;
+using SFA.DAS.AANHub.Application.Common.Validators.RequestedByMemberId;
+using SFA.DAS.AANHub.Application.Mediatr.Responses;
+using SFA.DAS.AANHub.Domain.Entities;
+using static SFA.DAS.AANHub.Domain.Common.Constants;
+
+namespace SFA.DAS.AANHub.Application.Partners
+{
+    public class CreatePartnerMemberCommand : CreateMemberCommandBase, IRequest<ValidatableResponse<CreatePartnerMemberCommandResponse>>, IRequestedByMemberId
+    {
+        public string UserName { get; set; } = null!;
+        public string Organisation { get; set; } = null!;
+        public Guid? RequestedByMemberId { get; set; }
+
+        public static implicit operator Member(CreatePartnerMemberCommand command) => new()
```

Comment:
Coverage missing for the operator?

Reply:
I think this is covered as part of `CreatePartnersMemberCommandHandlerTests`

Reply:
Same here, explicit test on the operator is expected with assertion on each field. Especially here as we have applied default values.

Reply:
Added here: https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/7d8c7338247663f8460af149a795a511a4610dd1/src/SFA.DAS.AANHub.Api.UnitTests/Controllers/PartnersControllerTests.cs#L79